src/native/src/core/nova_renderer.h

```diff
@@ -7,12 +7,13 @@
 
 #include <memory>
 
-#include "../interfaces/iwindow.h"
+#include "interfaces/iwindow.h"
+#include "mc/mc_objects.h"
+
 #include "nova.h"
-#include "../mc/mc_objects.h"
+#include "types.h"
```

Comment:
I didn't it looks like at some point i moved the imports around. mc_objects.h is above a few lines, right below iwindow.h